bundles/net.tourbook.common/src/net/tourbook/common/MyTourbookNotificationPopup.java

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (C) 2023 Frédéric Bard
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
+ *******************************************************************************/
+package net.tourbook.common;
+
+import net.tourbook.common.util.StringUtils;
+
+import org.eclipse.jface.layout.GridDataFactory;
+import org.eclipse.jface.notifications.AbstractNotificationPopup;
+import org.eclipse.jface.resource.ImageDescriptor;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Text;
+
+class MyTourbookNotificationPopup extends AbstractNotificationPopup {
+
+   private ImageDescriptor IMAGEDESCRIPTOR;
+
+   private String          _title;
+   private String          _text;
+
+   MyTourbookNotificationPopup(final Display display,
+                               final ImageDescriptor imageDescriptor,
+                               final String title,
+                               final String text) {
+      super(display);
+      IMAGEDESCRIPTOR = imageDescriptor;
```

Comment:
A private field should be named similar like `_imageDescriptor`

Reply:
fixed